crypto-conditions/src/main/java/org/interledger/cryptoconditions/RsaSha256Fulfillment.java

```diff
@@ -59,8 +59,7 @@ public RSAPublicKey getPublicKey() {
   /**
    * Returns a copy of the signature used in this fulfillment.
    *
-   * @return A {@link byte[]} containing the signature for this fulfillment.
-   *
```

Comment:
See my comment below - I think the newline should be kept between `@return` and `@deprecated`. Below is a better example.